docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -206,18 +219,33 @@ pyroscope.write "write_job_name" {
 
 ### Exposing pprof endpoints
 
-If you don't use `http.DefaultServeMux`, you can register `/debug/pprof/*` handlers to your own `http.ServeMux`
+If you don't use `http.DefaultServeMux`, you can register `/debug/pprof/*` handlers to your own `http.ServeMux`:
+
 ```go
 var mux *http.ServeMux
 mux.Handle("/debug/pprof/", http.DefaultServeMux)
 ```
+
 Or, if you use gorilla/mux:
+
 ```go
 var router *mux.Router
 router.PathPrefix("/debug/pprof").Handler(http.DefaultServeMux)
 ```
 
 ## References
+
+### Grafana Alloy
```

Comment:
These links below will all be in one line... is that on purpose?

Reply:
I'll fix that. Unintended.